{
  "id": "MAL-2024-10019",
  "summary": "Malicious code in ethreum (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-10-16T14:40:52Z",
  "published": "2024-10-16T14:40:52Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-10-16T14:40:52Z",
        "import_time": "2024-10-24T00:56:59.003679923Z",
        "versions": [
          "1.0.0"
        ],
        "id": "RLMA-2024-08246",
        "source": "reversing-labs",
        "sha256": "311fdb5bc7d6e8a524933ceec4f0d765c12e42169dc1e0bedde39d2ee643b62c"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "ethreum",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/ethreum"
      },
      "versions": [
        "1.0.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/ethreum/MAL-2024-10019.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}